### DISCUSSION TITLE
Manual/UIE-uxml-element-Button

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#676] [Notes]
# Manual/UIE-uxml-element-Button

https://docs.unity3d.com/Manual/UIE-uxml-element-Button.html

**Comment by Chaussurre** [comment]
The example given uses `uxmlButton.RegisterCallback<MouseUpEvent>((evt) => action());` to link the event to the action.
However, that means the event will be triggered if the click is released above the button, no matter where the click started. Most buttons in UIs usually only trigger if the mouse clicked and released both on the same button. That behavior can be achieved by using `Button.clicked` instead. For example:
`uxmlButton.clicked += action;`
While RegisterCallback is a method existing on all visual elements, clicked is a property unique to buttons, and is the proper way to use them.
